src/components/interior-left-nav/_interior-left-nav.scss

```diff
@@ -297,11 +297,11 @@
     transition: background-color 300ms $carbon--standard-easing,
       width 300ms $carbon--standard-easing;
     cursor: pointer;
-    background-color: rgba(#3d70b2, 0.1);
+    background-color: rgba($icon-01, 0.1);
```

Comment:
- [ ] Change to hex and override with `body.bx--v9-theme`  or find a token due to the css custom properties issue or other solution.